src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -6875,7 +6881,10 @@ static void F2<T>(S<T> x2)
 }";
 
             var comp = CreateCompilation(new[] { source, UnscopedRefAttributeDefinition }, parseOptions: TestOptions.Regular10);
-            comp.VerifyEmitDiagnostics();
+            comp.VerifyEmitDiagnostics(
+                // (8,28): error CS9063: UnscopedRefAttribute can only be applied to 'out' parameters, 'ref' and 'in' parameters that refer to 'ref struct' types, and instance methods and properties on 'struct' types other than constructors and 'init' accessors.
```

Comment:
It feels like use of this attribute pre-C# 11 should cause a FeatureNotSupported error referring to ref fields, scoped modifier, or some other relevant message. Obviously doesn't need to happen in this PR.